### ISSUE TITLE
PySide2 not available as backend

### ISSUE THREAD

**Issue by Vizmuth** [glumpy/glumpy#237]
Glumpy supports PyQt5 and PySide backend, why not PySide2? I searched for the reason, but couldn't find a single comment about it. Is there any technical or licensing issue for that?

**Comment by rougier** [comment]
No, it's just I'm not used to it. If you want to contribute a PySide2 backend that would be great. I think it is not that difficult if you know PySide2, just have a look at other backends.
